### ISSUE TITLE
Use Apertium for Translations

### ISSUE THREAD

**Issue by Nic-Wow1** [Billy-S/blockysurvival#11]
Why? The currently used Google Translator is proprietary and does not  support free software. [Apertium](https://www.apertium.org/) is a fully FLOSS translation platform.

**Comment by luk3yx** [comment]
Good idea, I'll see if I can add it to my bot, I had to disable my bot's auto-translations because of rate limiting.
What are the system requirements, and is there an online API for it?

**Comment by luk3yx** [comment]
I cannot find a web-based API (unless someone else wants to host it, I don't think I have the resources to).

Is there any HTTPS API (`https://example.com/translate/hola` → `hello`)?

**Comment by Sires0** [comment]
Ain't it https://www.apertium.org/index.por.html?dir=<L-x>-<L-y>&q=<txt>#translation
L-x = language from, L-y = language to, txt = what to translate

**Comment by Sires0** [comment]
the por in index.por is cuz the website for me was in portuguese

**Comment by luk3yx** [comment]
It seems to be `https://www.apertium.org/apy/translate?q=<text>&langpair=<from>|<to>`, and auto-detection is on `https://www.apertium.org/apy/identifyLang?q=<text>`.

@Sires0's link is the interactive user-facing translator webpage.

I am currently working on a more lightweight version of my bot, I'll want to see if I can add it to that as well.
